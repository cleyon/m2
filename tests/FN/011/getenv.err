m2:getenv.m2:3:Environment variable 'BAR' not defined:BAR is '@getenv BAR@'
