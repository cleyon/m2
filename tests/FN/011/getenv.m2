FOO is '@getenv FOO@'
SBAR is '@sgetenv SBAR@'
BAR is '@getenv BAR@'
