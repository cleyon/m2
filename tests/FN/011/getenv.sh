M2="$1"
file="$2"
FOO=QUUX ${M2} ${file}
