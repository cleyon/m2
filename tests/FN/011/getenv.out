FOO is 'QUUX'
SBAR is ''
